service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -345,6 +347,11 @@ public String getFurtherDirectionsText() {
         return Optional.ofNullable(orderFurtherDirections).map(FurtherDirections::getDirections).orElse("");
     }
 
+    @JsonIgnore
+    public String getExclusionClauseText() {
+        return Optional.ofNullable(orderExclusionClause).map(ExclusionClause::getExclusionClause).orElse("");
```

Comment:
Do you prefer this or the ternary
```suggestion
        return orderExclusionClause != null ? orderExclusionClause.getExclusionClause() : "";
```

Reply:
I guess it's taste! I don't mind either, it's a simple guard anyway

Reply:
Yeah, just wanted to get opinions on what people preferred, don't mind either way